hack/monitor_prow.py

```diff
@@ -17,16 +17,26 @@
 RESET = exec_cmd('tput', 'sgr0')
 CLEAR = exec_cmd('tput', 'clear')
 
+
+def run_oc(args):
+    command = ['oc', '--loglevel', '10', '--namespace', 'ci']
+    command.extend(args)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Any reason why not to use `subprocess.run` ([subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run))?  Would end up being a little bit pythonic:

```
try:
  process = subprocess.run(command, capture_output=True, check=True)
except subprocess.CalledProcessError as exc:
  print(exc.stderr.decode('utf-8')
  raise

return process.stdout.decode('utf-8')
```